frontend/src/routes/manage/shared.tsx

```diff
@@ -491,30 +485,25 @@ const DEFAULT_SORT_DIRECTION = "DESCENDING";
 
 /** Helper functions to read URL query parameters and convert them into query variables */
 type QueryVars = {
-    limit: number;
-    offset: number;
+    page: number;
     direction: SortDirection;
 }
 export const parsePaginationAndDirection = (
     queryParams: URLSearchParams,
     defaultDirection: SortDirection = "DESCENDING",
 ): QueryVars => {
-    const limitParam = queryParams.get("limit");
-    const limit = limitParam ? parseInt(limitParam, 10) : LIMIT;
-
     const pageParam = queryParams.get("page");
-    const page = pageParam ? parseInt(pageParam, 10) : 1;
-    const offset = Math.max(0, (page - 1) * limit);
+    const page = pageParam ? Math.max(parseInt(pageParam), 1) : 1;
```

Comment:
This doesn't deal with non-integer params. Passing `page=fasdfs` will result in `parseInt` returning a `NaN`, which poisons everything. I would deal with unparsable strings the same way as with non-existent params: use 1 as default.